examples/modify-pin/README.md

```diff
@@ -0,0 +1,60 @@
+# Setting and modifying PIN codes
+
+This example shows you the functionality and usage of the following four commands:
+* [set_pin](#set_pin-command) - Sets a new PIN code
```

Comment:
I would propose using the term "PIN value" instead of "PIN code" as it can be everything like a password and does not need to be a number